src/backend/TrafficCourts/TrafficCourts.Arc.Dispute.Service/Models/DriversLicence.cs

```diff
@@ -23,7 +23,8 @@ public static string WithCheckDigit(string driversLicence)
                 sum += (driversLicence[i] - '0') * _digitWeights[i];
             }
 
-            char checkDigit = Convert.ToChar((sum % 11) + '0');
+            // Calculate the check digit by taking the last digit of the modulus. (TCVP-2829)
+            char checkDigit = Convert.ToChar((sum % 11) % 10 + '0');
```

Comment:
This is some serious code kungfu. Great work.

Reply:
I really wanted this version :-)

```cs
		Vector<int> weights = new Vector<int>(new int[] { 1, 2, 5, 3, 6, 4, 8, 7 });
		Vector<int> digits = new Vector<int>(new int[] { 0, 1, 6, 9, 0, 0, 2, 8 });
		int checkSum = Vector.Sum(digits * weights) % 11 % 10;
		char checkDigit = Convert.ToChar('0' + checkSum);
```